crates/bevy_pbr/src/render/fast_math.wgsl

```diff
@@ -0,0 +1,43 @@
+#define_import_path bevy_pbr::fast_math
+
+#import bevy_render::maths::PI;
+
+// Reference: XEngine
+// https://github.com/ShawnTSH1229/XEngine/blob/main/Source/Shaders/FastMath.hlsl
+
+// 4th order polynomial approximation
+// 4 VGRP, 16 ALU Full Rate
+// 7 * 10^-5 radians precision
+// Reference : Handbook of Mathematical Functions (chapter : Elementary Transcendental Functions), M. Abramowitz and I.A. Stegun, Ed.
+fn fast_acos(x: f32) -> f32 {
```

Comment:
Fyi something similar already exists in ssao_utils.wgsl.

Reply:
oh I didn't notice lol. The polynomial version seems a little more accurate but I don't need that too much. I'll just merge the ssao_utils stuff into fast_math then, except the multibounce stuff.